nsot/models/resource.py

```diff
@@ -281,6 +276,21 @@ def _resource_name(self):
     def _purge_attribute_index(self):
         self.attributes.all().delete()
 
+    @classmethod
+    def get_concrete_field_names(self):
+        """Returns a list of the concrete field names as strings
+        defined on the resource model"""
+
+        # Grabbing the concrete fields for this Resource
+        concrete_field_names = []
+        # The below call can be replaced with .get_concrete_fields_with_model()
```

Comment:
Can you please add `TODO` to this comment?

Reply:
Done.